lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -602,7 +603,7 @@ def _section_send_email(course, access):
         # xblock rendering.
         request_token=uuid.uuid1().get_hex()
     )
-    email_editor = fragment.content
+    email_editor = HTML(fragment.content)
```

Comment:
@nedbat: I think it is better for this HTML() call to be in Python, closer to where the actual HTML snippet is being declared.  Upon further review of the code, it may make sense to push this even deeper into `wrap_xblock()`, or even deeper still into `wrap_fragment()`.  
When looking into this code, I realized it would take some time to determine if it really is sending safe HTML.  For example, I doubt that the [urls in this line](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/xblock_utils.py#L228) are safe. 

In this [blog post](http://reinout.vanrees.org/weblog/2014/11/14/6python-templating.html), it is written:

> Django also uses markupsafe one-directionally

What does this mean?  Does it mean if we used these same calls from Django Templates they wouldn't be known to be already safe?
 
FYI: @douglashall